src/validation/AuthValidation.ts

```diff
@@ -0,0 +1,31 @@
+import * as yup from "yup";
+
+export const signUpSchema = yup.object({
+  username: yup.string().required("First name is a required field."),
+  firstName: yup.string().required("First name is a required field."),
+  lastName: yup.string().required("Last name is a required field."),
+  email: yup
+    .string()
+    .email("Please make sure entered email is valid.")
+    .required("Email is a required field"),
```

Comment:
Add email case-insensitive validation
```suggestion
  email: yup
    .string()
    .email("Please make sure entered email is valid.")
    .required("Email is a required field")
    .test(
         'existsCheck',
         'Email already exists',
         value => !lowerEmails.includes(value.toLowerCase())
      ),
```

Reply:
what is lowerEmails in this case?

Reply:
email field

Reply:
Yup was throwing error. We can handle this in upcoming PRs. Raising an issue for the same.